### PR TITLE
create a Jobs XML file from the configuration

### PR DESCRIPTION
**Pull request by marcosmc**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Feature

**What is the new behavior (if this is a feature change)?**
Create a Jobs XML file from the configuration.

### CODE REVIEW

**Comment by mathbagu** [comment]
@marcosmc I'm not convinced that this is good approach. Your proposal is too Jaxb oriented, without Jaxb. I think we don't need to create a complete data model for inputs and outputs.

From powsybl point of view, Dynawo will have these inputs:
- a IIDM network
- a list of dynamic models associated to each static equipments
- some parameters
I propose to start with a `DynawoContext` class with the IIDM network and the DynamicSimulation parameters. We'll complete this class later, with all the data will need.

In this PR, we only need a `JobsXml` class that have one static public method `void write(Path path, DynawoContext)` to generate the content. So I propose to remove all the classes you create in the model package and the DynawoCollectionXmlFile class.

For unit test, I propose to:
- assert that the generated file is valid regarding the XSD
- assert that the generated file is equivalent to the reference one.
To do that, you have to put in the test resources the jobs.xsd file from dynawo release, and a reference file. I suggest to create also a utility test class because we'll do exactly these validation for all the inputs.

To sum up, we should have:
- DynawoContext: the context of the simulation
- JobsXml: a utility class to generate a Jobs file from a DynawoContext
- DynawoTestUtil: a utility class to shared test function used in unit tests
- jobs.xsd and jobs.xml: test resources
- JobsXmlTest: the test class

**Comment by marcosmc** [comment]
Understood, if that's okay with you, I start development on a new branch and we'll remove the current branch and the PR
